### PR TITLE
Search 273 (Electron bindings for attachment search)

### PR DESCRIPTION
**Pull request by keerthi16**
### Electron bindings for attachment search.

- Included new library
- Implementation for attachment search

@VikasShashidhar @jay-k & @serkanmulayim Please review. Thanks

### CODE REVIEW

File: js/search/search.js [hunk 1]
```diff
@@ -326,9 +326,10 @@ class Search {
      * @param {String} searchQuery
      * @param {Array} senderId
      * @param {Array} threadId
+     * @param {String} fileType
      * @returns {string}
      */
-    static constructQuery(searchQuery, senderId, threadId) {
+    static constructQuery(searchQuery, senderId, threadId, fileType) {
 
         let query = "";
```
Comment (VikasShashidhar) [inline js/search/search.js]: Can you refactor this to something like "searchText" as it's confusing with all the other names like searchQuery, etc.?

File: js/search/search.js [hunk 2]
```diff
@@ -54,6 +54,7 @@ class Search {
      * @param key
      */
     constructor(userId, key) {
+        console.time('Decrypting');
```
[the remark this answers is not in the document]
Comment (keerthi16) [reply js/search/search.js]: @serkanmulayim Needs this for measuring the performance. We can remove this when its get merged to master.

**Comment by VikasShashidhar** [comment]
@serkanmulayim If this is good to go, I'll merge. Please let me know.

**Comment by serkanmulayim** [comment]
LGTM

**Pull request merged**